config/firstboot/gb-firstboot

```diff
@@ -9,7 +9,7 @@ backup_dir=$HOME/.config/sway/config.d/backup
     sleep 2
     for i in XXX "Waiting for Internet connection" XXX 10;
     do echo $i; done
-    until /usr/bin/ping -q -c 1 flathub.org
+    until /usr/bin/curl -s --max-time 5 https://flathub.org > /dev/null
```

Comment:
Why? What’s wrong with `ping`? Establishing the whole HTTPS connection, just to check whether the server is up, seems a bit expensive.